tooling/package-maven-plugin/src/main/java/org/apache/camel/quarkus/maven/UpdateDocExtensionsListMojo.java

```diff
@@ -127,6 +127,11 @@ public void execute() throws MojoExecutionException, MojoFailureException {
                             final DataFormatModel delegate = (DataFormatModel) m.delegate;
                             delegate.setName("bindy");
                         }
+                        // TODO: Fix typo in ES REST metadata. Remove for Camel 3.3.0.
```

Comment:
Would you please file a followup issue for this?

Reply:
Yep - once this is merged, I'll raise one :+1: